pkg/client/ocm/config.go

```diff
@@ -63,6 +63,10 @@ func (c *OCMConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.ClusterLoggingOperatorAddonID, "cluster-logging-operator-addon-id", "", "The name of the cluster logging operator addon. An empty string indicates that the operator should not be installed")
 }
 
+func (c *OCMConfig) CheckIfCredentialsAreProvided() bool {
```

Comment:
Function that just returns a boolean, are usually used to answer to a specific question with a yes or no. IMO this would be better called something like `HasCredentials`, 'CredentialsExists`, `AreCredentialsProvided` or something like that.